libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/room/MatrixRoom.kt

```diff
@@ -99,11 +99,11 @@ interface MatrixRoom : Closeable {
 
     suspend fun inviteUserById(id: UserId): Result<Unit>
 
-    suspend fun canInvite(): Result<Boolean>
+    suspend fun canUserInvite(userId: UserId): Result<Boolean>
 
-    suspend fun canSendStateEvent(type: StateEventType): Result<Boolean>
+    suspend fun canUserSendState(userId: UserId, type: StateEventType): Result<Boolean>
 
-    suspend fun canSendEvent(type: MessageEventType): Result<Boolean>
+    suspend fun canUserSendMessage(userId: UserId, type: MessageEventType): Result<Boolean>
```

Comment:
Not sure that the new parameter `userId` will be useful for those 3 methods, but we will see.